lib/protocol/schema/definitions.capnp

```diff
@@ -37,7 +37,8 @@ struct Schema {
 }
 
 struct DealInfo {
-    id @0 :UInt64;
-    selectorPath @1 :Text;
-    cid @2 :Text;
+    cid @0 :Text;
```

Comment:
Changed the definition to return `cid`, dag `size`, `created`, and `isPermanent`

Reply:
I am slightly confused by removal of `selectorPath` because it seems like `cid` is a hot layer construct. We should be able to get any file that was uploaded by its CID as long as it is in the hot layer. If there data were to be retrieved from a storage provider wouldn't that require selector path to fetch data from the deal?